app/[locale]/tag/listing-tags.tsx

```
@@ -0,0 +1,126 @@
+        return arr
+          .sort(
+            (a, b) =>
+              (b.updatedAt?.getTime?.() || 0) - (a.updatedAt?.getTime?.() || 0)
+          )
+          .reverse();
+      case "date-asc":
+        return arr.sort(
+          (a, b) =>
+            (a.updatedAt?.getTime?.() || 0) - (b.updatedAt?.getTime?.() || 0)
+        );
+      case "popularity":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)